lib/helper/Puppeteer.js

```diff
@@ -248,9 +248,15 @@ class Puppeteer extends Helper {
       this.debugSection('Session', 'cleaning cookies and localStorage');
       await this.clearCookie();
     }
-    await this.executeScript('localStorage.clear();').catch((err) => {
-      if (!(err.message.indexOf("Storage is disabled inside 'data:' URLs.") > -1)) throw err;
-    });
+    const currentUrl = await this.grabCurrentUrl();
+
+    if (currentUrl !== 'about:blank') {
+      await this.executeScript('localStorage.clear();').catch((err) => {
+        if (!(err.message.indexOf("Storage is disabled inside 'data:' URLs.") > -1)) throw err;
+      });
+    } else {
+      this.debugSection('Session', 'cannot clear localstorage, because current url is "about:blank"');
```

Comment:
Ok, what's the difference between `data:` and `about:` here?
Why do we have 2 ways of handling this?